app/components/editable_champ/editable_champ_component.rb

```diff
@@ -34,7 +34,7 @@ def html_options
         }.merge(champ_component.input_group_error_class_names)
       ),
       data: { controller: stimulus_controller, **data_dependent_conditions, **stimulus_values }
-    }
+    }.merge(champ_component.fieldset_error_opts)
```

Comment:
Pour éviter un futur bug qui écraserait des attributs : 

```suggestion
    }.deep_merge(champ_component.fieldset_error_opts)
```